src/ralph/api/serializers.py

```diff
@@ -188,6 +190,13 @@ def build_field(self, *args, **kwargs):
             field_kwargs.setdefault('style', {})['base_template'] = 'input.html'
         return field_class, field_kwargs
 
+    def save(self):
```

Comment:
No, ex. `BaseObjectLicenceViewSet` and `BaseObjectLicenceSerializer` used in `save_serializer_class` - i assume that there is more cases like that.